tree/ntuple/inc/LinkDef.h

```diff
@@ -18,12 +18,12 @@
 #pragma link C++ nestedclasses;
 
 // Support for auto-loading in the RNTuple tutorials
-#pragma link C++ class ROOT::Experimental::Detail::RFieldBase-;
-#pragma link C++ class ROOT::Experimental::Detail::RFieldBase::RSchemaIterator-;
-#pragma link C++ class ROOT::Experimental::RVectorField-;
-#pragma link C++ class ROOT::Experimental::RNTupleReader-;
-#pragma link C++ class ROOT::Experimental::RNTupleWriter-;
-#pragma link C++ class ROOT::Experimental::RNTupleModel-;
+#pragma link C++ class ROOT::Experimental::Detail::RFieldBase;
```

Comment:
Those should be ```+``` [Without the '+' it request the old I/O scheme .. if the classes had a Streamer member function (i.e. had a ClassDef macro)]

Reply:
Actually, I think the correct way would be to remove dictionary creation for the current RNTuple classes entirely.  The only problem is that then cling does not load `libROOTNTuple` automatically and all the the tutorial need a `R__LOAD_LIBRARY(ROOTNTuple)` statement.  How can we make sure that autoloading works?